get-started/index.md

```diff
@@ -67,42 +67,41 @@ packageserver-8cccc99dd-g7lkh       1/1     Running   0          3m3s
 Use the search bar to find the **Red Hat build of Cryostat** catalog item.
 {% include howto_step.html
   summary="Select the Cryostat Operator and click the Install button"
-  image-name="cryostat-operatorhub-install.png"
+  image-name="`cryostat-operatorhub-install.png`"
 %}
 Choose your Operator installation mode:
-1. In **All Namespaces** installation mode, the Cryostat Operator instance will watch for `Cryostat` or
-`ClusterCryostat` Custom Resources (`CR`s) created in any Namespace and create corresponding Cryostat instances.
-2. In the **A specific namespace** installation mode, you must also select an installation Namespace, and the Cryostat
-Operator instance will only watch for `Cryostat` or `ClusterCryostat` instances created in that same Namespace.
+1. In **All Namespaces** installation mode, the **Cryostat Operator** instance will watch for **Cryostat** or
+**ClusterCryostat** Custom Resources (`CR`s) created in any Namespace and create corresponding Cryostat instances.
```

Comment:
Maybe "Namespace" -> "`Namespace`"? It seems like **bold** is used to highlight names of specific things, but `monospace` is used for `CR` as a Kubernetes resource type. I'm not sure if this is being consistently applied though - maybe CR should also be bolded instead of monospaced.

Should the "Cryostat" in "Cryostat instances" at the end of this line be bold?

Reply:
I think I missed the Cryostat in the last line. I will also bold the `CR`